com.unity.render-pipelines.universal/Documentation~/urp-shaders/urp-shaderlab-pass-tags.md

```diff
@@ -0,0 +1,24 @@
+# URP ShaderLab Pass tags
+
+This section contains descriptions of URP-specific ShaderLab Pass tags.
+
+## URP Pass tags: LightMode
+
+The value of this tag determines whether Unity executes the Pass during a given frame, when during the frame Unity executes the Pass, and what Unity does with the output.
```

Comment:
This sentence seems somewhat confusing.

`whether Unity executes the Pass during a given frame, when during the frame Unity executes the Pass` I'm not sure what "when ruding the frame" means.

`and what Unity does with the output` confused about this as well. Which output are we talking about?

Reply:
I've copied this sentence from the new doc on Shaders (which was reviewed), I'll see if we should change it there.

How about this version:
"The value of this tag lets the pipeline determine which Pass to use when executing different parts of the Render Pipeline."